README.md

```diff
@@ -45,160 +88,88 @@ To run a specific unittest:
-to match how the VA operates in production and staging environments. If you use
-a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections
-from the Docker instance to your host on the required ports.
-
-By default Boulder runs the legacy ACME v1 API endpoint on port `4000` and the
-ACME v2 API endpoint on port `4001`.
+Boulder's default VA configuration (`test/config/va.json`) is configured to connect to port 5002 to validate HTTP-01 challenges and port 5001 to validate TLS-SNI-01/TLS-ALPN-01 challenges. If you want to solve challenges with a client running on your host you should make sure it uses these ports to respond to validation requests, or update the VA configuration's `portConfig` to use ports 80 and 443 to match how the VA operates in production and staging environments. If you use a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections from the Docker instance to your host on the required ports.
 
-If a base image changes (i.e. `letsencrypt/boulder-tools`) you will need to rebuild
-images for both the boulder and bhsm containers and re-create them. The quickest way
-to do this is with this command:
+If a base image changes (i.e. `letsencrypt/boulder-tools`) you will need to rebuild images for both the boulder and bhsm containers and re-create them. The quickest way to do this is with this command:
 
     ./docker-rebuild.sh
 
-Working with a client:
-----------------------
 
-Check out the Certbot client from https://github.com/certbot/certbot and follow the setup instructions there. Once you've got the client set up, you'll probably want to run it against your local Boulder. There are a number of command line flags that are necessary to run the client against a local Boulder, and without root access. The simplest way to run the client locally is to source a file that provides an alias for certbot (`certbot_test`) that has all those flags:
+### Working with Certbot
+
+Check out the Certbot client from https://github.com/certbot/certbot and follow their setup instructions. Once you've got the client set up, you'll probably want to run it against your local Boulder. There are a number of command line flags that are necessary to run the client against a local Boulder, and without root access. The simplest way to run the client locally is to source a file that provides an alias for certbot (`certbot_test`) that has all those flags:
 
     source ~/certbot/tests/integration/_common.sh
     certbot_test certonly -a standalone -d example.com
 
-Your local Boulder instance uses a fake DNS server that returns 127.0.0.1 for
-any query, so you can use any value for the -d flag. You can also override that
-value by setting the environment variable FAKE_DNS=1.2.3.4
+Your local Boulder instance uses a fake DNS resolver that returns 127.0.0.1 for any query, so you can use any value for the -d flag. If you want to use another DNS resolver you can by setting the environment variable FAKE_DNS=1.2.3.4.
```

Comment:
The "If you want to use another..." sentence is also inaccurate for the reason described above.